bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,77 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
+    extract_documentation)
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class DocumentationStyleBear(LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/7sfk3i9oxs1ixg2ncsu3pym0u'
+    CAN_DETECT = {'Documentation'}
+
+    def run(self, filename, file, docstyle: str, language: str):
+        """
+        Checks for certain in-code documentation styles.
+
+        It currently checks for the following style:
+
+        ::
+
+            hello
+            :param x:
+                4 space indent
+            :return:
+                also 4 space indent
+                following lines are also 4 space indented
+        """
+        for doc_comment in extract_documentation(file, language, docstyle):
+            metadata = iter(doc_comment.parse())
+
+            # Assuming that the first element is always the only description.
+            new_metadata = [next(metadata)]
+
+            for m in metadata:
+                stripped_desc = list(map(str.lstrip, m.desc.splitlines(True)))
+                stripped_desc[0] = (('\n' + ' ' * 4 + stripped_desc[0])
+                                    if not stripped_desc[0] == ''
+                                    else stripped_desc[0])
+                new_desc = ''.join(('\n' if not line.endswith('\n') else '') +
+                                   (' ' * 4 if line != '' else '') + line
+                                   for line in stripped_desc)
```

Comment:
Yeah, could I do this later, this is something entirely different to the documentation style.